src/frontend/src/views/ProjectDetailsV2.tsx

```diff
@@ -382,11 +382,14 @@ const Home = () => {
               }`}
             >
               {import.meta.env.MODE === 'development' && (
-                <div className="fmtm-absolute fmtm-top-16 fmtm-left-4 fmtm-z-50">
-                  <Button
-                    btnText="Toggle Console"
-                    btnType="secondary"
-                    onClick={() => setShowDebugConsole(!showDebugConsole)}
+                <div className="fmtm-block sm:fmtm-hidden fmtm-absolute fmtm-top-6 fmtm-left-16 fmtm-z-50">
```

Comment:
I'll merge this PR and test 👍 

Just one note: could the debug console menu for development be moved to a higher level, maybe into App.jsx or something? So it can potentially be used on any page for debugging

Reply:
@spwoodcock I think the debugconsole works on all pages.  I have only added to project details page since the right side menu is not accessible on mobile devices. 
Also, when consoled on slice, reducers it throws page crash error

Reply:
Got you, thanks for clarifying 👍